apps/src/p5lab/AnimationTab/AnimationListItem.jsx

```diff
@@ -80,15 +84,28 @@ class AnimationListItem extends React.Component {
     }, 200);
   }
 
-  onSelect = () => this.props.selectAnimation(this.props.animationKey);
+  onSelect = () => {
+    if (this.props.currentAnimationType === 'background') {
```

Comment:
Can we use the `CURRENT_ANIMATION_TYPE` enum to avoid hard-coding `'background'` here?